pkgs/tools/games/steamtinkerlaunch/default.nix

```
@@ -16,13 +16,13 @@
 
 stdenv.mkDerivation rec {
   pname = "steamtinkerlaunch";
-  version = "12.12";
+  version = "0-unstable-2024-04-14";
 
   src = fetchFromGitHub {
     owner = "sonic2kk";
     repo = pname;
```

```suggestion
    repo = "steamtinkerlaunch";
```

Please also remove `rec`.